### PR TITLE
Feature/9453 document workflows

### PR DESCRIPTION
**Pull request by supriyaaddagada**
This PR is to review the documentation for the branching and terraform workflows

## Changes
\prime-reportstream\prime-router\docs\workflows\branching.md
\prime-reportstream\prime-router\docs\workflows\Terraform.md
\prime-reportstream\prime-router\docs\workflows\Terraform_Workflows.md

## Linked Issues
-  #9453

### CODE REVIEW

File: prime-router/docs/workflows/Terraform.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      uses: peter-evans/create-pull-request@v4
+      with:
+        commit-message: 'Terraform Plan'
+        branch: 'terraform-plan'
+        base: 'main'
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+</div>
+The previous GitHub Action example runs the following steps:
+1.	GitHub Action runs on pushes to the master branch
```
Comment (jeremy-page) [inline prime-router/docs/workflows/Terraform.md:52]: Would you please make this a numbered list? It shows as one line if you look at it as it is - https://github.com/CDCgov/prime-reportstream/blob/feature/9453-DocumentWorkflows/prime-router/docs/workflows/Terraform.md

**Review by jeremy-page — APPROVED** [review]
Awesome work! I put a request in for some formatting changes but those are trivial. This is great, thank you!

File: prime-router/docs/workflows/Terraform.md (hunk 1 above)
Comment (supriyaaddagada) [reply prime-router/docs/workflows/Terraform.md]: Recommended Formatting changes are applied

File: prime-router/docs/workflows/Terraform.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    # from the terraform.tfplan file in the branch
+    - name: Terraform Apply
+      run: terraform apply "terraform.tfplan"
+The previous GitHub Action runs the following steps:
+1.	GitHub Action runs on pushes to the master branch.
+2.	Repository code is checked out.
+3.	Cloud provider is authenticated. In this example, Microsoft Azure is authenticated against.
+4.	Terraform is setup using the hashicorp/setup-terraform action.
+5.	The Terraform init command is run to initialize the Terraform project.
+6.	The Terraform apply command is run, passing in the terraform.tfplan file to tell it what Terraform Plan to apply.
```
[the remark this answers is not in the document]
Comment (supriyaaddagada) [reply prime-router/docs/workflows/Terraform.md]: Recommended Formatting changes are applied

**Pull request merged**